tests/api/test_checkout.py

```diff
@@ -434,6 +435,55 @@ def test_checkout_available_shipping_methods(
     assert data["availableShippingMethods"] == [{"name": shipping_method.name}]
 
 
+@pytest.mark.parametrize(
+    "expected_price_type, expected_price, display_gross_prices",
+    (("gross", 13, True), ("net", 10, False)),
+)
+def test_checkout_available_shipping_methods_with_price_displayed(
+    expected_price_type,
+    expected_price,
+    display_gross_prices,
+    monkeypatch,
+    api_client,
+    checkout_with_item,
+    address,
+    shipping_zone,
+    site_settings,
+):
+    shipping_method = shipping_zone.shipping_methods.first()
+    taxed_price = TaxedMoney(net=shipping_method.price, gross=Money(13, "USD"))
```

Comment:
I assume that you can just create TaxedMoney(net=Money(10, "USD", gross=Money(13, "USD")).
Especially that you assume that `shipping_method.price` is `net` whic can be missleading for someone who checks the flow of taxes